lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install

```diff
@@ -108,3 +108,28 @@ function dosomething_mbp_update_7003(&$sandbox) {
   }
 
 }
+
+/**
+ * Creates production entry in message_broker_producer for user profile updates
+ */
+ function dosomething_mbp_update_7004(&$sandbox) {
+
+   $affiliate = variable_get('message_broker_producer_application_id', 'US');
```

Comment:
is this not the same as the country code?  DRY https://github.com/DoSomething/dosomething/blob/dev/lib/modules/dosomething/dosomething_settings/dosomething_settings.module#L74